### PR TITLE
Add High Priority Feature

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add toggleable priority to cards
* Add confirmation dialog service for easily producing confirmation dialogs
* Fix toast error styling for smaller messages
*

### CODE REVIEW

File: alcs-api/src/application/application.service.ts [hunk 1]
```diff
@@ -46,31 +46,20 @@ export class ApplicationService {
   async createOrUpdate(
     application: Partial<Application>,
   ): Promise<Application> {
-    let applicationEntity = await this.applicationRepository.findOne({
+    const existingApplication = await this.applicationRepository.findOne({
       where: { fileNumber: application.fileNumber },
     });
 
-    // TODO: replace with AutoMapper
```
Comment (mhuseinov) [inline alcs-api/src/application/application.service.ts:53]: 👍  nice cleanup

File: alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.html [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+    </span>
+  </div>
+  <div class="center">
+    <div class="card-state">{{ currentCard.paused ? 'Paused' : 'Active' }}</div>
+    <div>
+      <button (click)="onTogglePriority()" mat-icon-button>
+        <span
+          [ngClass]="{
+            'high-priority': currentCard.highPriority,
+            'low-priority': !currentCard.highPriority
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.html:15]: should this be just regular priority?

Comment (mhuseinov) [reply alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.html:15]: Is there a chance of getting more than 2 priorities? If yes maybe a priority code will work better here

**Review by mhuseinov — APPROVED** [review]

File: alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.html (hunk 2 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.html:15]: Renamed the css class to empty-priority and filled-priority to better match what they mean

File: alcs-api/src/application/application.service.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-api/src/application/application.service.ts:53]: I started to add AutoMapper but realized it was from Application Entity to Application Entity to Object.assign was all that was needed.

**Pull request merged**